lib/project_types/script/messages/messages.rb

```diff
@@ -142,6 +142,9 @@ module Messages
           script_upload_cause: "Fail to upload script.",
           script_upload_help: "Try again.",
 
+          script_too_large_cause: "The size of your Wasm binary file is too large.",
+          script_too_large_help: "Make sure that it is less than %{max_size}.",
```

Comment:
You could probably just say `It must be less than {max size}`.

Reply:
@kwringe wrote this. I'm fine with either copy though. @kwringe what do you think of this?